core/src/core/pmmr/pmmr.rs

```diff
@@ -302,7 +302,7 @@ where
 	/// Walks all unpruned nodes in the MMR and revalidate all parent hashes
 	pub fn validate(&self) -> Result<(), String> {
 		// iterate on all parent nodes
-		for n in 1..(self.last_pos + 1) {
+		for n in 1..self.last_pos + 1 {
```

Comment:
Personal preference here is the original version reads better, even if the parens are not required.

Reply:
👍